ddtrace/contrib/internal/django/patch.py

```diff
@@ -147,7 +148,12 @@ def cursor(django, pin, func, instance, args, kwargs):
         tags = {"django.db.vendor": vendor, "django.db.alias": alias}
         tags.update(getattr(conn, "_datadog_tags", {}))
 
-        pin = Pin(service, tags=tags, tracer=pin.tracer)
+        # Calling ddtrace.pin.Pin(...) with the `tracer` argument generates a deprecation warning.
+        # Remove this if statement when the `tracer` argument is removed
+        if pin.tracer is ddtrace.tracer:
+            pin = Pin(service, tags=tags)
```

Comment:
With multiple tracers gone, is there a chance that we can perhaps drop the concept of `Pin`s entirely?